### ISSUE TITLE
Output tripping markup validation error in spec-prod: CSS: “border-radius”: only “0” can be a “unit”

### ISSUE THREAD

**Issue by inexorabletash** [speced/bikeshed#2733]
Seen in https://github.com/w3c/IndexedDB/actions/runs/7171892684/job/19527849573?pr=415 (and the PR itself wouldn't have caused this).
```
  Validating index.html...
  $ pnpm add vnu-jar
  + vnu-jar 23.4.11
  $ java -jar "/home/runner/work/_actions/w3c/spec-prod/v2/node_modules/.pnpm/vnu-jar@23.4.11/node_modules/vnu-jar/build/dist/vnu.jar" --also-check-css index.html
    "file:/home/runner/work/IndexedDB/IndexedDB.w3c/index.html":442.20-442.20: error: CSS: “border-radius”: only “0” can be a “unit”. You must put a unit after your number.
  ❌  Not so good... please fix the issues above.
```
bikeshed/bikeshed/dfnpanels/dfnpanels.css has:
```css
border-radius: 3;
```
... which might be tripping this, not sure yet - my dev machine is offline at the moment so I can't minimize easily.

**Comment by inexorabletash** [comment]
Processing IndexedDB TOT locally with a freshly updated Bikeshed:
```
~/spec/IndexedDB [main]$ cat -n index.html | egrep 'border-radius'
   445      border-radius: 3;
   529          border-radius: .25em;
   833  code.highlight { padding: .1em; border-radius: .3em; }
   834  pre.highlight, pre > code.highlight { display: block; padding: 1em; margin: .5em 0; overflow: auto; border-radius: 0; }
```
... which is close enough to the error line (442) and corresponds to that `dfnpanels.css` entry .

**Comment by inexorabletash** [comment]
Change was introduced in https://github.com/speced/bikeshed/commit/1d552322ab1b8912d718d0dd60280334002293b3 which was between semver 3.14.5 and 4.0.0

FYI @dlaliberte and @tabatkins

**Comment by tabatkins** [comment]
Fixed in 4.0.1.
